hyungjoonYoon/.env.sample

```
@@ -0,0 +1,9 @@
+PORT = YOUR PORT NUMBER SHOULD BE HERE! 
+
+TYPEORM_CONNECTION = mysql
+TYPEORM_HOST = host_number
+TYPEORM_USERNAME = root
+TYPEORM_PASSWORD = password
+TYPEORM_DATABASE = westagram
+TYPEORM_PORT = 3306
+TYPEORM_LOGGING =TRUE
```

해당 정보를 통해 외부, 제 3자에게 노출되었을 때 데이터베이스 내부 정보가 유출될 가능성이 높습니다.

다른 개발자가 해당 프로젝트에 참여하여 동일한 `.env` 파일을 구성할 수 있도록 명시해주는 역할을 하는 파일입니다. 

`sample`이라는 파일명과 부합하도록 수정해주세요!
```suggestion
TYPEORM_CONNECTION = RDBMS_TYPE
TYPEORM_HOST = HOST_NUMBER
TYPEORM_USERNAME = USERNAME
TYPEORM_PASSWORD = PASSWORD
TYPEORM_DATABASE = DATABASE_NAME
TYPEORM_PORT = PORT_NUMBER
TYPEORM_LOGGING = BOOLEAN
```